src/main/java/com/epam/tc/gdi/hw1/site/pages/components/simple/HeaderNavigation.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.gdi.hw1.site.pages.components.simple;
+
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Link;
+
+
+public class HeaderNavigation extends Section {
+
+    @FindBy(css = "a[href=\"index.html\"]")
```

Comment:
It's easier to use @FindBy(css = "a[href='index.html']") and do not escape symbols (and easy to understand)

Reply:
Done